src/commands/upload-coverage.yml

```diff
@@ -8,5 +8,6 @@ description: >
   Upload test coverage to codecov.io
 
 steps:
-  - run: ./node_modules/.bin/nyc report --reporter text-lcov > coverage.lcov
-  - codecov/upload
+  - run:
+      name: Run and Upload CodeCov
```

Comment:
`- run: echo "code coverage not implemented"` so it doesn't still have the CodeCov name in it (someone will be searching repos, I'm sure!)